stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/Channel.kt

```diff
@@ -45,12 +45,23 @@ public fun Channel.getDisplayName(
     @StringRes fallback: Int = R.string.stream_ui_channel_list_untitled_channel,
 ): String {
```

Comment:
What do you think about placing `Channel.getDisplayName` into the `ui-common` module?

That way we could use it as is both in UI Components and in Compose and would not have to duplicate code.